thirdparty/cyrus-sasl/patches/sasl-auth-zimbra-2.1.28.patch

```diff
@@ -0,0 +1,1576 @@
+Index: cyrus-sasl-2.1.26/saslauthd/auth_zimbra.c
```

Comment:
2.1.28?

Reply:
we haven't changed this patch, anyway it doesn't matter, it is just path to directory used for creation of patch.